components/Badge/Badge.tsx

```diff
@@ -5,13 +5,26 @@ import { Slot } from '@radix-ui/react-slot';
 export const COLORS = ['gray', 'red', 'blue', 'green', 'neon', 'orange', 'purple'] as const;
 type COLOR_VALUES = typeof COLORS[number];
 
+const getColorBadgeStyles = (color: COLOR_VALUES) => ({
+  bc: `$${color}6`,
+  color: `$${color}10`,
+});
+
 type ColorVariants = Record<COLOR_VALUES, { bc: string; color: string }>;
 const colorVariants: ColorVariants = COLORS.reduce(
+  (variants, color) => ({
+    ...variants,
+    [color]: getColorBadgeStyles(color),
+  }),
+  {} as ColorVariants
+);
+
+const interactiveColorVariants: ColorVariants = COLORS.reduce(
   (variants, color) => ({
     ...variants,
     [color]: {
-      bc: `$${color}6`,
-      color: `$${color}10`,
+      ...getColorBadgeStyles(color),
+      '&:focus-visible': { outline: `2px solid $${color}9` },
```

Comment:
What if we make border color similar accross components? Card and switch has the same `primary` color as border, but badge is not, wdyt?

![image](https://user-images.githubusercontent.com/70909035/164710283-8f4e3b61-4b53-4d5a-948c-e687dbfdad0f.png)

![image](https://user-images.githubusercontent.com/70909035/164710648-28f638c6-1849-4012-b3bd-122650d4da35.png)

Reply:
@gndz07 I don't have a strong opinion there. I just checked that primary color was fine with all the color variants.

I agree applying similar outline effect across components makes sense. I would simply apply the same `2px` outline on all those.

WDYT?